pkg/osbuild/systemd_unit_create_stage.go

```diff
@@ -177,3 +229,112 @@ func NewSystemdUnitCreateStage(options *SystemdUnitCreateStageOptions) *Stage {
 		Options: options,
 	}
 }
+
+// GenSystemdMountStages generates a collection of
+// org.osbuild.systemd.unit.create stages with options to create systemd mount
+// units, one for each mountpoint in the partition table.
+func GenSystemdMountStages(pt *disk.PartitionTable) ([]*Stage, error) {
+	mountStages := make([]*Stage, 0)
+	unitNames := make([]string, 0)
+
+	genOption := func(ent disk.FSTabEntity, path []disk.Entity) error {
+		fsSpec := ent.GetFSSpec()
+		fsOptions, err := ent.GetFSTabOptions()
+		if err != nil {
+			return err
+		}
+
+		var filename string
+		var options *SystemdUnitCreateStageOptions
+		device := fmt.Sprintf("/dev/disk/by-uuid/%s", strings.ToLower(fsSpec.UUID))
```

Comment:
I prefer `filepath.Join` which suits fine here.

Reply:
Done.

Reply:
I think you missed the one here and did the one below?